tools/lkl/Makefile

```diff
@@ -42,15 +42,15 @@ conf: $(OUTPUT)Makefile.conf
 $(OUTPUT)Makefile.conf $(OUTPUT)include/kernel_config.h $(OUTPUT)/kernel.config: Makefile.autoconf
 	$(call QUIET_AUTOCONF, headers)$(MAKE) -f Makefile.autoconf -s
 
--include $(OUTPUT)Makefile.conf
+include $(OUTPUT)Makefile.conf
```

Comment:
TIL: https://www.gnu.org/software/make/manual/html_node/Include.html

> If an included makefile cannot be found in any of these directories it is not an immediately fatal error; processing of the makefile containing the include continues. Once it has finished reading makefiles, make will try to remake any that are out of date or don’t exist.

Nice, thank you @rodionov !